### DISCUSSION TITLE
Stuck FPS in launching games POCO F3

### DISCUSSION THREAD

**Discussion by elvetics13** [skyline-emu/skyline#1500] [Q&A] [answered]
Can u guys help me about why games that compatible like SoS: Friends of mineral town keep stuck in launching?
![Screenshot_2022-06-09-07-06-04-576_skyline emu](https://user-images.githubusercontent.com/106851985/172737405-18a42d63-9c92-45c9-bfbc-9d7149862a29.jpg)

**Comment by mateoalfaro** [comment]
Probably just not compatible yet

**Comment by Crytonics** [comment, marked as answer]
> Can u guys help me about why games that compatible like SoS: Friends of mineral town keep stuck in launching? ![Screenshot_2022-06-09-07-06-04-576_skyline emu](https://user-images.githubusercontent.com/106851985/172737405-18a42d63-9c92-45c9-bfbc-9d7149862a29.jpg)

For info to how to run game go see this: https://github.com/skyline-emu/skyline-games-list/issues/269
```
"When select name it will crash, because skyline still not add input keyboard feature, so i must playing using save files."
```

**Reply by Mocitradon** [reply]
I'm experiencing the same issue with Mario Strikers & Soldiers

**Reply by mateoalfaro** [reply]
Incompatible games

**Reply by Crytonics** [reply]
Some games need save files to run or update.
